procbuild/listener.py

```diff
@@ -22,31 +20,31 @@ class Listener:
     
     Attributes:
     ------------
-    ctx: zmq.asyncio.Context, the main context for the listener class
-    
-    prefix: str, the prefix listened for by zmq sockets
-    
-    socket: zmq.socket, the socket for listening to 
-    
-    queue: asyncio.Queue, the queue for holding the builds
-    
-    dont_build: set, unique collection of PRs currently in self.queue
+    ctx: zmq.asyncio.Context 
+        main context for the listener class
+    socket: zmq.socket
+        the socket for listening to 
+    queue: asyncio.Queue
```

Comment:
I'm not sure how forgiving `numpydoc` is, but you might need spaces after the parameter name.